_content/v2-preview/java/_api.md

```diff
@@ -105,6 +121,7 @@ client.onMessage("my-uno", "led", new BiConsumer<String, Object>() {
         System.out.println("Message: " + devId + " " + data);
 });
 ```
+  > When a field is specified, `cb.data` holds the `String` representation of the field value.
```

Comment:
Yes but since java is strongly typed, the received value will be passed to the user as a String. Should this be sent as a byte array ?